src/diagnostics/logPointsClient.ts

```diff
@@ -78,15 +78,15 @@ export class KuduLogPointsDebuggerClient extends LogPointsDebuggerClientBase imp
 
     public async startSession(siteName: string, affinityValue: string, publishCredential: WebSiteModels.User): Promise<CommandRunResult<IStartSessionResponse>> {
         await this.uploadSshClient(siteName, affinityValue, publishCredential);
-        return this.makeCallAndLogException<IStartSessionResponse>(siteName, affinityValue, publishCredential, "curl -X POST http://localhost:32923/debugger/session");
+        return this.makeCallAndLogException<IStartSessionResponse>(siteName, affinityValue, publishCredential, "curl -s -S -X POST http://localhost:32923/debugger/session");
```

Comment:
What does "-s" (lower case) try to achieve? 
```
-s, --silent        Silent mode (don't output anything)
```

Wouldn't it mean no output can be seen?

Reply:
We tested these parameters with Mike, and we get to see the stderr without any timing information, which is what we want to show the user

Reply:
As long as you tested and saw that we have the correct JSON response from stdout. I am fine with it.

Reply:
Yes, this show what we want